### ISSUE TITLE
[Bug]: Adding second server to Beszel

### ISSUE THREAD

**Issue by MasterPuffin** [coollabsio/coolify#4599]
### Error Message and Logs

So I set up Beszel via the Coolify template and it monitors the server it is running on just fine. However now I want to add a different server.
I am running the Beszel agent on the second server with the following docker compose file

services:
  beszel-agent:
    image: "henrygd/beszel-agent"
    container_name: "beszel-agent"
    restart: unless-stopped
    network_mode: host
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      PORT: 45876
      KEY: "ssh-ed25519 XXXX"

In the Beszel UI I set my Service Name as the Host/IP. Connect To Predefined Network is also activated for both services in Coolify.
However the Beszel UI can't connect to the second server. Any idea why?

### Coolify Version

v4.0.0-beta.379

### Are you using Coolify Cloud?

No (self-hosted)

**Comment by peaklabs-dev** [comment]
I think this is most likely not a bug in Coolify, but rather a configuration issue/support question. So I will close this, please open a support question on our Discord server https://coollabs.io/discord. If this is a bug with use we can reopen this issue.

**Comment by MasterPuffin** [comment]
I already submitted a support question on Discord https://discord.com/channels/459365938081431553/1318008844542017557/1318008844542017557
However I don't think it's an configuration issue. As far as I understand Docker the second host should be reachable when both containers are exposed to the Coolify network. That's why I suspect there might be an issue with the Coolify network.

**Comment by peaklabs-dev** [comment]
The problem is probably your network configuration, because Coolify uses containers that are in isolated separate networks from the host system, so if you have the Coolify server and the second server in the same network, that will still not work, as a container is in its own network. You could try adding the host network to the Beszel container to make it work.

**Comment by MasterPuffin** [comment]
When I add `network_mode: host` to the docker-compose Beszel returns an 502 or 503 error when trying to access the web ui.
I tried the option in Coolify `Connect To Predefined Network`.
I also added a custom external network to all docker-compose files like this
```
networks:
  beszel_network:
    driver: bridge
    external: true
```
for hostnames I tried <container id>, beszel-agent-<container id>, and <container name>, none worked. 
I also found nothing how Coolify handles networks between servers.

**Comment by liolocs** [comment]
I also had this issue where I installed the UI and the agent on server A but then I wanted to add another system (my localhost) which is server B in this case. @MasterPuffin Use the Binary setup instead of the Docker setup to add your other system and it'll work.

**Comment by MasterPuffin** [comment]
> I also had this issue where I installed the UI and the agent on server A but then I wanted to add another system (my localhost) which is server B in this case. [@MasterPuffin](https://github.com/MasterPuffin) Use the Binary setup instead of the Docker setup to add your other system and it'll work.

Yes adding it with the binary works, however you loose the simplicity of docker networks and have to open the port

**Comment by FranckKe** [comment]
I did not look into this usecase when I added the service as the single server setup was enough for me. I can update the service documentation when a more official stance on handling external networks and communication between Coolify instances has been made.
